.changes/changed/2761.md

```diff
@@ -0,0 +1 @@
+Renamed `ConsensusParametersProvider` to `ChainStateInfoProvider` because it is now providing more than just info about consensus parameters.
```

Comment:
Hmm... this is technically a breaking change since we're renaming a public facing type. But I doubt this type being used anywhere by anyone so I think it could fly under the radar without being marked as breaking. I don't know how strict we'll be about this so feel free to decide yourself, but I think personally I'd lean on marking this as breaking.